package.json

```diff
@@ -9,7 +9,9 @@
   "scripts": {
     "test": "jest",
     "test:watch": "jest --watch",
-    "start": "nodemon node app.js"
+    "debug": "nodemon node app.js",
```

Comment:
```suggestion
    "dev": "nodemon node app.js",
```

Reply:
What do you think of changing the names from degug and start to be startproduction and startdev